modules/nw-ipfailover-configuration.adoc

```diff
@@ -41,7 +41,45 @@ $ oc adm policy add-scc-to-user privileged -z ipfailover
 ----
 $ oc adm policy add-scc-to-user hostnetwork -z ipfailover
 ----
+
+. {rh-openstack-first} only: Complete the following steps to make the failover VIP addresses reachable on {rh-openstack} ports:
++
+.. Use the {rh-openstack} CLI to find one IP address in the `allowed_address_pairs` parameter of your {rh-openstack} cluster.
++
+[source,terminal]
+----
+$ openstack port show <cluster_name> -c allowed_address_pairs
+----
++
+[source,terminal,subs="attributes,quotes"]
```

Comment:
If this an output it might be worth labeling it with a title.

Reply:
:heavy_check_mark: